### PR TITLE
Add callback processing

### PR DESCRIPTION
**Pull request by yarikgenza**

### CODE REVIEW

File: packages/react-mobile-navigation-core/src/components/Interpolation.native.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    onClose();
+  }
+
+  render() {
+    const { children, pageHeight, pageWidth } = this.props;
+    const { isVisible } = this.state;
+    const pageHeightNew = pageHeight;
+    const pageWidthNew = pageWidth - (2 * MODAL_MARGIN);
+    return isVisible ? (
+      <TouchableWithoutFeedback onPress={this.closeOnBackdrop}>
```
Comment (rtkhanas) [inline packages/react-mobile-navigation-core/src/components/Interpolation.native.js]: Why do we need this `this.closeOnBackdrop`? This function is used on a modal to let user close it down on a dark transparent area click. It seems like not needed here.

File: packages/react-mobile-navigation-core/src/components/Interpolation.native.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  }
+
+  close() {
+    const { onPageCloseDone } = this.props;
+    if (isFunction(onPageCloseDone)) {
+      onPageCloseDone();
+    }
+  }
+
+  closeOnBack() {
```
Comment (rtkhanas) [inline packages/react-mobile-navigation-core/src/components/Interpolation.native.js]: Is this function use anywhere?

Comment (yarikgenza) [inline packages/react-mobile-navigation-core/src/components/Interpolation.native.js]: No. removed.

File: packages/react-mobile-navigation-core/src/components/Interpolation.native.js (hunk 1 above)
Comment (yarikgenza) [inline packages/react-mobile-navigation-core/src/components/Interpolation.native.js]: Removed, thanks.

**Pull request merged**